dropwizard/service/build.gradle.kts

```diff
@@ -140,6 +141,8 @@ tasks.named<Test>("test").configure {
   if (System.getenv("AWS_REGION") == null) {
     environment("AWS_REGION", "us-west-2")
   }
+  environment("POLARIS_BOOTSTRAP_POLARIS_ROOT_CLIENT_ID", "test-admin")
+  environment("POLARIS_BOOTSTRAP_POLARIS_ROOT_CLIENT_SECRET", "test-secret")
```

Comment:
Where do these get used?

Reply:
In `DropwizardServerManager`. I initially wanted to run the server in a separate JVM, but that proved to be too cumbersome. I can switch back to getting secrets from the local services by direct java calls, if you prefer.

Reply:
It seems to be hardcoded currently https://github.com/apache/polaris/blob/f6d2a8c9bcfa03e45b8a5d136bea68a43c0313b5/dropwizard/service/src/test/java/org/apache/polaris/service/dropwizard/it/DropwizardServerManager.java#L108

Also would these credentials be valid? I thought Polaris generates random credentials.

Reply:
Hardcoded, yes, is that a concern?

The env. variables enable user-provided bootstrap secrets per #422

Reply:
Ah I was missing context on #422, thanks!